snowshu/logger.py

```diff
@@ -48,7 +54,8 @@ def set_log_level(self, level: str) -> None:
             if handler != self.file_handler:
                 handler.setLevel(level)
 
-    def remove_all_handlers(self, logger: logging.Logger) -> None:
+    @staticmethod
+    def remove_all_handlers(logger: logging.Logger) -> None:
```

Comment:
This is another common fix. 
`self` not being used in the method --> convert method to a staticmethod (solution)